calculators/items/HotPotatoBook.js

```diff
@@ -0,0 +1,49 @@
+const { APPLICATION_WORTH } = require('../../constants/applicationWorth');
+
+class HotPotatoBookCalculation {
+    constructor({ calculation, itemData, prices, price }) {
+        this.calculation = calculation;
+        this.itemData = itemData;
+        this.prices = prices;
+        this.price = price;
+        // this.calculate();
```

Comment:
Tehnically we could call this, so there would be no need for [this](https://github.com/Altpapier/SkyHelper-Networth/blob/revamp-duckysolucky/calculators/ItemNetworthCalculator.js#L113-L116).

Reply:
Yeah, no need for a constructor. Harder to maintain and not need if we use static methods.